src/explore-education-statistics-frontend/src/components/Feedback.tsx

```diff
@@ -0,0 +1,188 @@
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import styles from '@frontend/components/Feedback.module.scss';
+import Button from '@common/components/Button';
+import classNames from 'classnames';
+import { useMemo, useState } from 'react';
+import FormProvider from '@common/components/form/FormProvider';
+import { Form, FormFieldTextArea } from '@common/components/form';
+import ButtonGroup from '@common/components/ButtonGroup';
+import feedbackService from '@frontend/services/feedbackService';
+import {
+  FeedbackRequest,
+  FeedbackResponse,
+} from '@common/services/types/feedback';
+import Yup from '@common/validation/yup';
+import { ObjectSchema } from 'yup';
+
+type BannerState = 'initial' | 'notUseful' | 'problemEncountered' | 'thanks';
+
+interface FormValues {
+  context?: string;
+  issue?: string;
+  intent?: string;
+}
+
+const feedbackLimit = 2000;
+
+export default function Feedback() {
+  const [bannerState, setBannerState] = useState<BannerState>('initial');
+  const [response, setResponse] = useState<FeedbackResponse | undefined>();
+
+  const handleUsefulFeedback = async () => {
+    await feedbackService.sendFeedback({
+      url: window.location.pathname,
+      userAgent: navigator.userAgent,
+      response: 'Useful',
+    });
+
+    setBannerState('thanks');
+  };
+
+  const cancel = () => {
+    setResponse(undefined);
+    setBannerState('initial');
+  };
+
+  const handleSubmit = async (values: FeedbackRequest) => {
+    await feedbackService.sendFeedback({
+      url: window.location.pathname,
+      userAgent: navigator.userAgent,
```

Comment:
Should probably truncate the user agent if it exceeds 250 characters to ensure that we don't get a server error (and subsequent submit error) if it can't be saved to the DB.